chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -246,48 +246,24 @@ public DeveloperDTO update(final DeveloperDTO updatedDev, final boolean doValida
             throw new EntityCreationException(msg);
         }
 
-        // determine if the status has been changed in most cases only allowed by ROLE_ADMIN but ROLE_ACB
-        // can change it to UnderCertificationBanByOnc
-        boolean currentStatusChanged = !currDevStatus.getStatus().getStatusName()
```

Comment:
I am just wondering where all this deleted code went... when I saw it initially I was expecting it to be moved into a permissions class. Did we just not need it?

Reply:
I think i got a little over zealous removing code.  I reverted the commit and re-coded the change.